bin/q.py

```diff
@@ -3297,106 +3297,103 @@ def initialize_command_line_parser(p, qrc_filename):
         print("Incorrect value '%s' for option %s in .qrc file %s (option type is %s)" % (
         e.actual_value, e.option, qrc_filename, e.option_type))
         sys.exit(199)
-    parser = OptionParser(prog="q",usage=USAGE_TEXT)
-    parser.add_option("-v", "--version", dest="version", default=False, action="store_true",
-                      help="Print version")
-    parser.add_option("-V", "--verbose", dest="verbose", default=default_verbose, action="store_true",
+    parser = ArgumentParser(prog="q",usage=USAGE_TEXT)
+    parser.add_argument("-v", "--version", action="store_true", help="Print version")
+    parser.add_argument("-V", "--verbose", default=default_verbose, action="store_true",
                       help="Print debug info in case of problems")
-    parser.add_option("-S", "--save-db-to-disk", dest="save_db_to_disk_filename", default=default_save_db_to_disk,
+    parser.add_argument("-S", "--save-db-to-disk", dest="save_db_to_disk_filename", default=default_save_db_to_disk,
                       help="Save database to an sqlite database file")
-    parser.add_option("-C", "--caching-mode", dest="caching_mode", default=default_caching_mode,
+    parser.add_argument("-C", "--caching-mode", default=default_caching_mode,
                       help="Choose the autocaching mode (none/read/readwrite). Autocaches files to disk db so further queries will be faster. Caching is done to a side-file with the same name of the table, but with an added extension .qsql")
-    parser.add_option("", "--dump-defaults", dest="dump_defaults", default=False, action="store_true",
```

Comment:
I saw that you've removed all dest="X" parameters. Am I assuming correctly that it's because it's auto-generated from the long-flag-name?

Reply:
Yes, `argparse` is smart enough to replace all the dashes in the long flag names into underscores